lib/video_info/providers/youtube.rb

```diff
@@ -79,14 +85,22 @@ def _default_url_attributes
         {}
       end
 
-      def _video_entry
-        data['entry']
+      def _video_snippet
+        data['items'][0]['snippet']
       end
 
-      def _video_media_group
-        data['entry']['media$group']
+      def _video_content_details
+        data['items'][0]['contentDetails']
       end
 
+      def _video_statistics
+        data['items'][0]['statistics']
+      end
+
+      # def _video_media_group
```

Comment:
I didn't notice that was commented out, or that this functionality was even available. Is there a spec for it? Because all of the specs pass except '#available?'